docs/agent_hocon_reference.md

```diff
@@ -239,27 +219,84 @@ Note: _We strongly recommend to **not** set secrets as values within any source
 These files tend to creep into source control repos, and it is **very** bad practice
 to expose secrets by checking them in.
 
-#### verbose
+#### class
+
+You can use the `class` key in two ways:
+
+#### 1. For supported providers (predefined in `default_llm_info.hocon`)
```

Comment:
Shouldn't 1 and 2 have another "#" for being a sub-heading under class?
That is: Class has 4x#  1. and 2. each have 5x#